tests/notification.spec.ts

```diff
@@ -13,32 +15,36 @@ import {
   dismissFirstNotificationFromList,
 } from "./helpers/notification";
 
-test("show notification after user action", async ({ page }) => {
-  const instance = randomInstanceName();
+let instance = randomInstanceName();
+let page: Page;
+test.beforeAll(async ({ browserName, browser }) => {
+  page = await browser.newPage();
+  instance = `${browserName}-${instance}`;
   await createInstance(page, instance);
-  await checkNotificationExists(page);
+});
+
+test.afterAll(async () => {
   await deleteInstance(page, instance);
+  await page.close();
+});
+
+test("show notification after user action", async () => {
+  await visitAndStartInstance(page, instance);
   await checkNotificationExists(page);
 });
 
-test("dismiss one notification", async ({ page }) => {
-  const instance = randomInstanceName();
-  await createInstance(page, instance);
-  await dismissNotification(page);
-  await deleteInstance(page, instance);
+test("dismiss one notification", async () => {
+  await visitAndStopInstance(page, instance);
   await dismissNotification(page);
 });
 
-test("auto hide notification after a timeout", async ({ page }) => {
-  const instance = randomInstanceName();
-  await createInstance(page, instance);
+test("auto hide notification after a timeout", async () => {
+  await visitAndStartInstance(page, instance);
   await page.waitForTimeout(5000);
```

Comment:
If we export the `HIDE_NOTIFICATION_DELAY` const in `toastNotificationProvider.tsx`, we can directly reference that here, so that we track any changes on that value:
```suggestion
  await page.waitForTimeout(HIDE_NOTIFICATION_DELAY);
```

Reply:
great idea, thanks!

Reply:
Had to revert this change unfortunately, during build time for tests we don't have scope of `react-components` module which is imported in the same file where the `HIDE_NOTIFICATION_DELAY` live. This causes a module not found error for any react components we import in `toastNotificationProvider`

Reply:
Wouldn't moving the const to a separate util file solve this? Then you would import it in both places (the test file and the `toastNotificationProvider`. Does it sound reasonable and likely to work?

Reply:
I thought about putting it in `util/notifications.tsx` but that file also imports from react-components :sweat: didn't think creating another new utils file was worth it.